samples/testing-frameworks/appium/client-side/csharp/biometric-sample-test/BitbarBiometricSample/TestBiometricsAndroid.cs

```diff
@@ -9,13 +9,13 @@ public class TestBiometricsAndroid : BaseBiometricTest
         public void InitDriver()
         {
             //Platform dependent
-            capabilities.AddAdditionalCapability("platformName", "Android");
-            capabilities.AddAdditionalCapability("deviceName", "Android Phone");
-            capabilities.AddAdditionalCapability("automationName", "Appium");
+            capabilities.AddAdditionalCapability("appium:platformName", "Android");
```

Comment:
according to https://appium.io/docs/en/2.0/guides/caps/
platformName should be without prefix